foundry.toml

```diff
@@ -6,9 +6,8 @@ optimizer-runs = 1_000
 src = 'contracts'
 out = 'out'
 libs = ['node_modules', 'lib']
-test = 'test/foundry'
+test = 'test'
 cache_path = 'cache_forge'
-remappings = []
 
 # Hedgey Profile
```

Comment:
The hedgey strategies tests required this, but it's no longer needed. It can be removed along with its usage here https://github.com/allo-protocol/allo-v2/blob/3ce91e28644a2f29109f2416bb5a5c8a89eed096/package.json#L11-L14